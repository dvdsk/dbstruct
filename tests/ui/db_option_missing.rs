use dbstruct::dbstruct;

#[dbstruct]
struct Test {
    field: Option<u8>,
}

fn main() {}
