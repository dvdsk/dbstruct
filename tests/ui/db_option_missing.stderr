error: No database backend specified to use as backend
 --> tests/ui/db_option_missing.rs:3:1
  |
3 | #[dbstruct]
  | ^^^^^^^^^^^
  |
  = help: specify a backend using #[dbstruct(db=sled)]
  = note: this error originates in the attribute macro `dbstruct` (in Nightly builds, run with -Z macro-backtrace for more info)
